### PR TITLE
Toner: Add README.md and Toner.png

### PR DESCRIPTION
**Pull request by r0neko**
Hello! This is a Google Code-In Task. I was told to make a pull request where I put the README file and a screenshot of the application. Hope you like it! Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Toner
+------------
+
+![Toner screenshot](Toner.png "Toner")
+
+Toner is a Haiku Application that generates tones for aligning levels in your playback system.
```
Comment (humdingerb) [inline README.md]: "Application" would be lower-case. But maybe simpler is even better:
> A tool to generate tones for aligning the levels in your playback system.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by r0neko** [comment]
I updated the README.md file.

File: README.md (hunk 1 above)
Comment (r0neko) [reply README.md]: Done! Thanks for your feedback!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Toner
+------------
+
+![Toner screenshot](Toner.png "Toner")
+
+Toner is a tool to generates tones for aligning levels in your playback system.
```
Comment (humdingerb) [inline README.md]: It's "generate". Also, since the heading is "Toner", it's OK to shorten to "A tool to generate..." I think.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Review by r0neko — COMMENTED** [review]
Updated it again.

File: README.md (hunk 2 above)
Comment (r0neko) [reply README.md]: Sorry... my keyboard it's a little bit crazy today. Fixed it...

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thank you. :)

**Pull request merged**